### ISSUE TITLE
Set up OBS builds for this project

### ISSUE THREAD

**Issue by siteshwar** [att/ast#520]
We should provide latest builds for different Linux distros through OBS.

**Comment by krader1961** [comment]
While I agree that this should be done on general principals I think it should be deferred until the use-after-free bugs are fixed. That is, ksh can be built and pass its unit tests using the native malloc implementation provided by the platform.

**Comment by siteshwar** [comment]
These use after free bugs have accumulated slowly over years. It's going to be interesting how many of them we can fix before next release.

**Comment by krader1961** [comment]
> These use after free bugs have accumulated slowly over years.

How did you conclude that? I have already shown that the ksh93u+ release had only one use-after-free bug based on the unit tests at that time. That was roughly six years ago. I don't think we have any way of knowing if the current use-after-free failures are due to one, two, or a multitude of changes made after the ksh93u+ release and whether they occurred in a short or a long period. I suspect that the use-after-free failures we are talking about were introduced by one, or a very small number, of changes. If they are due to a single change then by definition they did not accumulate over several years. Even if they were due to more than one change it seems likely, given that they all involve the name/val logic, that they occurred over a relatively short period. Which is why I really, __really__, want to have access to the change history prior to the creation of this Github project.

**Comment by siteshwar** [comment]
> How did you conclude that? I have already shown that the ksh93u+ release had only one use-after-free bug based on the unit tests at that time. 

I think you only see a subset of these use after free issues on your local machine. Currently all non-API test cases are failing if you use `meson test --setup=malloc` on Travis. Our priority should be to fix the issues that were introduced after last stable release.

**Comment by krader1961** [comment]
I tested the 93u code with a patched vmalloc to poison freed buffers on all the VMs I have setup at that time. Too, of all the machines I've been testing on my main macOS system produces the most use-after-free failures using the current master branch. So even if that had been the only machine I performed that test on it should have yielded the most errors. The 93u release quite clearly contains far fewer bugs of this nature. But you don't have to take my word for it. I posted the diff for the 93u code to poison freed buffers in one of the issues so it's easy enough to verify. If you consider building ksh using the legacy build system easy :-)

Regardless, I still maintain that until the current code exhibits no new bugs when tested with `--setup=malloc` we should not publish prebuilt binaries.
